web-server/v0.3/js/jschart.js

```diff
@@ -13,10 +13,421 @@
+
+	  This example would request the available data every 5
+	  seconds and maintain a history of 300 samples.
+
+       k. raw_data_sources[]
+
+          The raw_data_sources option is an arbitrarily sized array
+          that contains links that should be appended to the table
+          associated with each chart.  This is typically used to
+          provide a way to access the raw tool output that the charted
+          data was generated with, hence the name.
+
+	  Example:
+
+	    { csvfiles: [ "data-files/processed.csv" ], raw_data_sources: [ "data-files/tool1.out", "data-files/tool2.out" ] }
+
+       l. threshold
+
+          The threshold option defines a value which is compared
+          against the maximum Y axis value for each dataset.  If the
+          threshold value is larger than the dataset's value then that
+          dataset is automatically hidden by default.  This provides a
+          mechanism which is used to filter out noise from tools that
+          produce large numbers of datasets.  At run time the user can
+          achieve the same functionallity through the UI and also
+          apply the threshold against the dataset average instead of
+          it's maximum value.  The UI controls do not require that any
+          options be provided in the code.
+
+	  Example:
+
+	    { ..., threshold: 5 }
+
+       m. sort_datasets
+
+          The sort_datasets option is a boolean value that determines
+          the order in which datasets are presented to the user.  By
+          default sort_datasets is enabled, unless live_update is used
+          (this is due to the requirement of consistent ordering
+          between the existing and new sample data).  If sort_datasets
+          is disabled by setting it to false then datasets are
+          presented in the order they are supplied to the library
+          (note: this was the default behavior prior to the addition
+          of sort_datasets).
+
+	  Example:
+
+	    { ..., sort_datasets: false }
+
+       n. x_min
+       o. x_max
+       p. y_min
+       q. y_max
+
+          These four options control the default axes domain of the
+          chart view port.  There are many different reasons for doing
+          this but the most common is probably to set a strict range
+          for a known set of values, such as a CPU usage graph which
+          will be a percentage between 0 and 100.
+
+	  Example:
+
+	    { ..., y_min: 0, y_max: 100 }
+
+       r. x_log_scale
+       s. y_log_scale
+
+          These two options are boolean values that control whether or
+          not the corresponding axis scale should be presented with a
+          logarithmic scale (instead of the default linear scale).
+          The one exception to this is if the 'timeseries' data model
+          is used, in that case the x_log_scale option is ignored.
+
+	  Example:
+
+	    { ..., y_log_scale: true }
+
+*/
+
```

Comment:
No, thank you, apparently it was quite needed.  The grammar doesn't surprise me, the spelling does.  I really thought I checked this before committing.